pyrelational/strategies/regression/bald_strategy.py

```diff
@@ -29,13 +29,30 @@ def __init__(
         self,
         temperature: float = 0.5,
     ):
+        """
+        :param temperature: parameter greater than 0 to divide scores before applying softmax.
+            A lower temperature parameter will give a peakier distribution of probabilities
```

Comment:
I had to look up if peakier is a proper description. In some circles it is! I learned something today.

Reply:
It is a weird word one for sure